### PR TITLE
chore(ollama): add documentation to self hosting ollama instance

### PR DESCRIPTION
**Pull request by namwoam**
Because

- We want to help users setup ollama instance and connect it to VDP.

This commit

- Add documentation with:
    - Install ollama
    - Expose ollama to local network
    - Connect ollama server to ollama client on VDP

### CODE REVIEW

File: ai/ollama/v0/.compogen/setup-hosting.mdx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#### Local Ollama Instance
+
+To set up an Ollama instance on your local machine, follow the instructions below:
+
+> Note: These instructions only work for Instill Core CE
```
Comment (GeorgeWilliamStrong) [inline ai/ollama/v0/.compogen/setup-hosting.mdx:5]:
```suggestion
<InfoBlock type="info" title="info"> 
  Please note that these instructions will only work for self-hosted **🔮 Instill Core**.
</InfoBlock>
```

File: ai/ollama/v0/.compogen/setup-hosting.mdx [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+1. Follow the tutorial on the official [GitHub repository](https://github.com/ollama/ollama) to install Ollama on your machine.
+2. Follow the instructions in the [FAQ section](https://github.com/ollama/ollama/blob/main/docs/faq.md) to modify the variable `OLLAMA_HOST` to `0.0.0.0`, then restart Ollama.
+3. Get the IP address of your machine on the local network.
+    - On Linux and macOS, open the terminal and type `ifconfig`.
+    - On Windows, open the command prompt and type `ipconfig`.
+4. Suppose the IP address is `192.168.178.88`, then the Ollama hosting endpoint would be `192.168.178.88:11434`.
+5. Enjoy fast LLM inference on your local machine with VDP's capability.
```
Comment (GeorgeWilliamStrong) [inline ai/ollama/v0/.compogen/setup-hosting.mdx:13]:
```suggestion
5. Enjoy fast LLM inference on your local machine and integration with 💧 Instill VDP.
```

File: ai/ollama/v0/README.mdx [hunk 3]
```diff
@@ -64,7 +64,19 @@ Provide text outputs in response to text/image inputs.
...
+
+> Note: These instructions only work for Instill Core CE
+
+1. Follow the tutorial on the official [GitHub repository](https://github.com/ollama/ollama) to install Ollama on your machine.
+2. Follow the instructions in the [FAQ section](https://github.com/ollama/ollama/blob/main/docs/faq.md) to modify the variable `OLLAMA_HOST` to `0.0.0.0`, then restart Ollama.
+3. Get the IP address of your machine on the local network.
+    - On Linux and macOS, open the terminal and type `ifconfig`.
+    - On Windows, open the command prompt and type `ipconfig`.
+4. Suppose the IP address is `192.168.178.88`, then the Ollama hosting endpoint would be `192.168.178.88:11434`.
+5. Enjoy fast LLM inference on your local machine with VDP's capability.
```
Comment (GeorgeWilliamStrong) [inline ai/ollama/v0/README.mdx:79]:
```suggestion
5. Enjoy fast LLM inference on your local machine and integration with 💧 Instill VDP.
```

**Review by GeorgeWilliamStrong — CHANGES_REQUESTED** [review]
Looks great, requested some minor changes 🙏

File: ai/ollama/v0/.compogen/setup-hosting.mdx (hunk 1 above)
[the remark this answers is not in the document]
Comment (namwoam) [reply ai/ollama/v0/.compogen/setup-hosting.mdx:5]: I think "CE" stands for community edition?

Comment (namwoam) [reply ai/ollama/v0/.compogen/setup-hosting.mdx:5]: Fixed

File: ai/ollama/v0/README.mdx (hunk 3 above)
Comment (namwoam) [reply ai/ollama/v0/README.mdx:81]: Fixed

**Pull request merged**